clustering/marshalling/protostream/src/main/java/org/wildfly/clustering/marshalling/protostream/LoadedClassField.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2020, Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags. See the copyright.txt file in the
+ * distribution for a full listing of individual contributors.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+
+package org.wildfly.clustering.marshalling.protostream;
+
+import java.io.IOException;
+import java.io.InvalidClassException;
+
+import org.infinispan.protostream.impl.WireFormat;
+import org.wildfly.security.manager.WildFlySecurityManager;
+
+/**
+ * A class field that marshals instances of {@link Class} using a {@link ClassResolver}.
```

Comment:
s/ClassResolver/ClassLoaderMarshaller ?

Reply:
Oops - just noticed that my IDE's javadoc validation settings were disabled.  Will enable and correct here and elsewhere.